build/Makefile

```diff
@@ -62,10 +62,10 @@ $(foreach t,$(DEFAULT_TARGETS),$(eval $(call per_target,$(t))))
 
 SUPPRESS_ERROR_OUTPUT ?= 2> /dev/null
 
-# We need to ensure bin exists for Travis CI builds
+# We need to ensure generated exists for Travis CI builds
```

Comment:
add / after generated to make it clear it's a directory

Reply:
Added